Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/DemoTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Aquality.Selenium.Browsers;
+using Aquality.Selenium.Template.NUnit.Constants;
+using Aquality.Selenium.Template.NUnit.Models;
+using Aquality.Selenium.Template.NUnit.Steps;
+using Aquality.Selenium.Template.Utilities;
+using NUnit.Framework;
+
+namespace Aquality.Selenium.Template.NUnit.Tests
+{
+    public class DemoTest : BaseTest
+    {
+        private const string FullPageSizeJS = "return document.body.scrollHeight";
```

Comment:
oh no, js should not be hardcoded like this and should not be used in the test class directly

Reply:
Tthe code was moved to a separate js file.